opentracing-noop/src/main/java/io/opentracing/noop/NoopSpan.java

```diff
@@ -19,7 +19,7 @@
 import java.util.Map;
 
 public interface NoopSpan extends Span {
-    static final NoopSpan INSTANCE = new NoopSpanImpl();
+    NoopSpan INSTANCE = new NoopSpanImpl();
```

Comment:
What's the reason for dropping the `static final`?

Reply:
There's none, I think. We need to restore those values.